README.md

```diff
@@ -1 +1,4 @@
-# opencraft-simple-theme
\ No newline at end of file
+# EDX Simple theme
+
+This theme is used in instances deployed by Ocim to theme OpenCraft client instances.
+This is used together with simple-theme role from edx-configuration repos.
```

Comment:
This can be deleted if the previous suggestion is accepted.

Reply:
:+1: